RecoTracker/TkSeedGenerator/interface/MultiHitGeneratorFromPairAndLayers.h

```diff
@@ -33,6 +33,16 @@ class MultiHitGeneratorFromPairAndLayers {
                         SeedingLayerSetsHits::SeedingLayerSet pairLayers,
                         std::vector<SeedingLayerSetsHits::SeedingLayer> thirdLayers) = 0;
 
+  virtual void hitTriplets(
+			   const TrackingRegion& region, 
+			   OrderedMultiHits & result,
+			   const edm::EventSetup & es,
+			   const HitDoublets & doublets,
+			   const RecHitsSortedInPhi ** thirdHitMap,
+			   const std::vector<const DetLayer *> & thirdLayerDetLayer,
+			   const int nThirdLayers)
+  {};
```

Comment:
Same comment as https://github.com/cms-sw/cmssw/pull/14083/files/fa87e30257849253e95ae9c485f2a9441f613490#r59851411 about declaring the method as pure virtual.